sidebars.js

```diff
@@ -90,118 +66,182 @@ module.exports = {
-      "area_registry_index",
-    ],
-  },
-  Core: {
-    "Development Workflow": [
-      "development_index",
-      "development_environment",
-      "development_submitting",
-      "development_guidelines",
-      "development_testing",
-      "development_catching_up",
-      "development_tips",
-    ],
-    "Building Integrations": [
-      "creating_component_index",
-      "creating_integration_file_structure",
-      "creating_integration_brand",
-      "creating_integration_manifest",
-      "config_entries_config_flow_handler",
-      "config_entries_options_flow_handler",
-      "configuration_yaml_index",
-      "dev_101_services",
-      "creating_platform_index",
-      "creating_component_generic_discovery",
-      "integration_fetching_data",
-      "integration_setup_failures",
-      "integration_events",
-      "integration_listen_events",
-      "network_discovery",
-      "bluetooth",
-    ],
-    "Development Checklist": [
-      "development_checklist",
-      "creating_component_code_review",
-      "creating_platform_code_review",
-      "integration_quality_scale_index",
-    ],
-    "Accessing the Core": [
-      "dev_101_hass",
-      "dev_101_events",
-      "dev_101_states",
-      "dev_101_config",
-    ],
-    Entities: [
-      "core/entity",
-      {
-        type: "autogenerated",
-        dirName: "core/entity",
-      },
-    ],
-    Platforms: [
-      {
-        type: "autogenerated",
-        dirName: "core/platform",
-      },
-    ],
-    Registries: [
-      "area_registry_index",
-      "device_registry_index",
-      "entity_registry_index",
-      "entity_registry_disabled_by",
-      "issue_registry_index",
-    ],
-    "Device Automations": [
-      "device_automation_index",
-      "device_automation_trigger",
-      "device_automation_condition",
-      "device_automation_action",
-    ],
-    Intents: [
-      "intent_index",
-      "intent_firing",
-      "intent_handling",
-      "intent_builtin",
-      "intent_conversation_api",
-    ],
-    Misc: ["development_validation", "development_typing", "instance_url"],
-  },
-  Misc: {
-    Introduction: ["misc"],
-    "Building a Python library": [
-      "api_lib_index",
-      "api_lib_auth",
-      "api_lib_data_models",
-    ],
-    asyncio: [
-      "asyncio_index",
-      "asyncio_101",
-      "asyncio_categorizing_functions",
-      "asyncio_working_with_async",
-    ],
-    "Maintainer docs": ["maintenance"],
-  },
+  Supervisor: [
+    "supervisor",
+    "supervisor/development",
+    "supervisor/debugging",
+    {
+      type: "category",
+      label: "Supervisor API",
+      items: [
+        "api/supervisor/endpoints",
+        "api/supervisor/models",
+        "api/supervisor/examples",
+      ],
+    },
+  ],
+  Core: [
+    {
+      type: "category",
+      label: "Architecture",
+      items: ["architecture/core", "architecture_components"],
+    },
+    {
+      type: "category",
+      label: "APIs",
```

Comment:
I'd move the APIs section down to the bottom as it's for experienced readers, not first visitors.

Reply:
I moved it up because I assumed a majority of people are not interested in developing _for_ Home Assistant but instead want to interface _with_ Home Assistant.

Reply:
Moved it down. Renamed section to External APIs. It's also linked from the frontend.